doc/api/repl.md

```diff
@@ -86,239 +135,319 @@ The special variable `_` (underscore) contains the result of the last expression
+`replServer.defineCommand()` method.
 
 ## repl.start([options])
 
-Returns and starts a `REPLServer` instance, that inherits from
-[Readline Interface][]. Accepts an "options" Object that takes
-the following values:
+* `options` {Object}
+  * `prompt` {String} The input prompt to display. Defaults to `> `.
+  * `input` {Readable} The Readable stream from which REPL input will be read.
+    Defaults to `process.stdin`.
+  * `output` {Writable} The Writable stream to which REPL output will be
+    written. Defaults to `process.stdout`.
+  * `terminal` {boolean} If `true`, specifies that the `output` should be
+    treated as a a TTY terminal, and have ANSI/VT100 escape codes written to it.
+    Defaults to checking the value of the `isTTY` property on the `output`
+    stream upon instantiation.
+  * `eval` {Function} The function to be used when evaluating each given line
+    of input. Defaults to an async wrapper for the JavaScript `eval()`
+    function.  An `eval` function can error with `repl.Recoverable` to indicate
+    the input was incomplete and prompt for additional lines.
+  * `useColors` {boolean} If `true`, specifies that the default `writer`
+    function should include ANSI color styling to REPL output. If a custom
+    `writer` function is provided then this has no effect. Defaults to the
+     REPL instances `terminal` value.
+  * `useGlobal` {boolean} If `true`, specifies that the default evaluation
+     function will use the JavaScript `global` as the context as opposed to
+     creating a new separate context for the REPL instance. Defaults to `false`.
+  * `ignoreUndefined` {boolean} If `true`, specifies that the default writer
+     will not output the return value of a command if it evaluates to
+     `undefined`. Defaults to `false`.
+  * `writer` {Function} The function to invoke to format the output of each
+     command before writing to `output`. Defaults to [`util.inspect()`][].
+  * `replMode` - A flag that specifies whether the default evaluator executes
+    all JavaScript commands in strict mode, default mode, or a hybrid mode
+    ("magic" mode.) Acceptable values are:
+    * `repl.REPL_MODE_SLOPPY` - evaluates expressions in sloppy mode.
+    * `repl.REPL_MODE_STRICT` - evaluates expressions in strict mode. This is
+      equivalent to prefacing every repl statement with `'use strict'`.
+    * `repl.REPL_MODE_MAGIC` - attempt to evaluates expressions in default
+      mode.  If expressions fail to parse, re-try in strict mode.
+
+The `repl.start()` method creates and starts a `repl.REPLServer` instance.
+
+## The Node.js REPL
+
+Node.js itself uses the `repl` module to provide its own interactive interface
+for executing JavaScript. This can used by executing the Node.js binary without
+passing any arguments (or by passing the `-i` argument):
 
- - `prompt` - the prompt and `stream` for all I/O. Defaults to `> `.
+```
+$ node
+Type '.help' for options.
```

Comment:
Yeah, it's not very helpful functionality at all IMO. I wonder if we should even mention it.